app/code/core/Mage/Core/Helper/Data.php

```diff
@@ -1001,4 +1001,12 @@ public function unEscapeCSVData($data)
         }
         return $data;
     }
+
+    /**
+     * @return bool
+     */
+    public function isFormKeyEnabled()
```

Comment:
```suggestion
    public function isFormKeyEnabled(): bool
```

Would use return/paremeter types for every new method.

Reply:
done, right, i've removed the docblock since it wouldn't have anything, not sure if that's the right way but I think so.

Reply:
I like doc blocks. Even if its redundant, its _easier_ to read ...

Reply:
mah, i'm not sure, anyway i've readded it